packages/main/src/plugin/container-registry.ts

```diff
@@ -1675,4 +1676,16 @@ export class ContainerProviderRegistry {
       this.telemetryService.track('buildImage', telemetryOptions);
     }
   }
+
+  async info(engineId: string): Promise<Info> {
+    let telemetryOptions = {};
+    try {
+      return this.getMatchingPodmanEngine(engineId).info();
+    } catch (error) {
+      telemetryOptions = { error: error };
+      throw error;
+    } finally {
+      this.telemetryService.track('info', telemetryOptions);
+    }
+  }
```

Comment:
hello, I think here we should prefix info to say it's a LibPodInfo because usually when there is no prefix it means it's coming from the compat API

Reply:
how does it work when you're using Docker engine ?